addons/turbowarp-player/addon.json

```
@@ -14,7 +14,7 @@
     },
     {
       "type": "info",
-      "text": "If the \"Replace player\" mode is selected, you can Ctrl+Click the button to open the project in a new TurboWarp tab.",
+      "text": "If the \"Replace player\" mode is selected, you can Ctrl+Click (on macOS Cmd+Click) the button to open the project in a new TurboWarp tab.",
```

```suggestion
      "text": "If the \"Replace player\" mode is selected, you can Ctrl+Click (Cmd+Click on macOS) the button to open the project in a new TurboWarp tab.",
```

I think the above is more readable, but I can't tell for sure because I'm not a native English speaker.

---

Speaking grammatically, this might work better as "on MacOS, Cmd+Click" or what WorldLanguages suggested.

---

Both are correct, but as a native English speaker I would use W_L's solution if I were writing this.